packages/mask/shared/index.ts

```diff
@@ -1,4 +1,4 @@
 export * from './messages'
 export * from './flags'
 export { InMemoryStorages, PersistentStorages } from './kv-storage'
-export * from './helpers/download'
```

Comment:
looks like you're containing changes from other PRs.

Reply:
It depend pr: https://github.com/DimensionDev/Maskbook/pull/5939 .